__tests__/app.test.js

```diff
@@ -234,6 +257,18 @@ describe("news api error handling test suite", () => {
         expect(response.body.msg).toBe("Please pass correct comment object!");
       });
   });
+  test("PATCH /api/articles/1 - status:400, responds with an error message if newVote object is missing required field", () => {
+    const updateVoteBy = {};
+    return request(app)
+      .patch("/api/articles/1")
+      .send(updateVoteBy)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe(
+          "Please pass correct object to update votes property!"
+        );
+      });
+  });
 });
```

Comment:
Great tests so far, but I feel you could have a few more! What if article doesn't exist? eg. `/api/articles/1000` or if the article isn't valid eg. `/api/articles/nonsense` Also, what if the value of inc_votes is not a number? eg. `{inc_votes: banana}`? These tests will make this airtight!